packages/contracts-bedrock/test/L1/OPContractsManager.t.sol

```diff
@@ -166,157 +164,86 @@ contract OPContractsManager_Upgrade_Harness is CommonTest {
 
-        // Reset the upgrader's code to the original code.
+        // Reset the upgrader to the original code.
         vm.etch(_delegateCaller, delegateCallerCode);
 
-        // Grab the new implementations.
-        address newPDGImpl = address(disputeGameFactory.gameImpls(GameTypes.PERMISSIONED_CANNON));
-        IPermissionedDisputeGame pdg = IPermissionedDisputeGame(newPDGImpl);
-        address newFDGImpl = address(disputeGameFactory.gameImpls(GameTypes.CANNON));
-        IFaultDisputeGame fdg = IFaultDisputeGame(newFDGImpl);
-
-        // Check that the PermissionedDisputeGame is upgraded to the expected version, references
-        // the correct anchor state and has the mipsImpl. Although Upgrade 15 doesn't actually
-        // change any of this, we might as well check it again.
-        assertEq(ISemver(address(pdg)).version(), "1.8.0");
-        assertEq(address(pdg.vm()), impls.mipsImpl);
-        assertEq(pdg.l2ChainId(), oldPDG.l2ChainId());
-
-        // If the old FaultDisputeGame exists, we expect it to be upgraded. Check same as above.
-        if (address(oldFDG) != address(0)) {
-            assertEq(ISemver(address(fdg)).version(), "1.8.0");
-            assertEq(address(fdg.vm()), impls.mipsImpl);
-            assertEq(fdg.l2ChainId(), oldFDG.l2ChainId());
-        }
-
-        // Make sure that the SystemConfig is upgraded to the right version. It must also have the
-        // right l2ChainId and must be properly initialized.
-        assertEq(ISemver(address(systemConfig)).version(), "3.7.0");
-        assertEq(impls.systemConfigImpl, EIP1967Helper.getImplementation(address(systemConfig)));
-        assertEq(systemConfig.l2ChainId(), l2ChainId);
-        DeployUtils.assertInitialized({ _contractAddress: address(systemConfig), _isProxy: true, _slot: 0, _offset: 0 });
-
-        // Make sure that the OptimismPortal is upgraded correctly.
-        assertEq(address(optimismPortal2.anchorStateRegistry()), address(asr));
-        DeployUtils.assertInitialized({
-            _contractAddress: address(optimismPortal2),
-            _isProxy: true,
-            _slot: 0,
-            _offset: 0
-        });
-
-        if (opcm.isDevFeatureEnabled(DevFeatures.OPTIMISM_PORTAL_INTEROP)) {
-            assertEq(ISemver(address(optimismPortal2)).version(), "5.0.0+interop");
-            assertEq(impls.optimismPortalInteropImpl, EIP1967Helper.getImplementation(address(optimismPortal2)));
-        } else {
-            assertEq(ISemver(address(optimismPortal2)).version(), "5.0.0");
-            assertEq(impls.optimismPortalImpl, EIP1967Helper.getImplementation(address(optimismPortal2)));
-        }
+        // We expect there to only be one chain config for these tests, you will have to rework
+        // this test if you add more.
+        assertEq(opChainConfigs.length, 1);
+
+        // Run the StandardValidator checks.
+        _opcm.opcmStandardValidator().validate(
```

Comment:
No more random checks, just use the Standard Validator and we can apply the exact same logic to each upgrade. Far simpler.